sklearn/datasets/olivetti_faces.py

```diff
@@ -113,16 +110,18 @@ def fetch_olivetti_faces(data_home=None, shuffle=False, random_state=0,
     data_home = get_data_home(data_home=data_home)
     if not exists(data_home):
         makedirs(data_home)
-    filepath = _pkl_filepath(data_home, TARGET_FILENAME)
+    filepath = _pkl_filepath(data_home, 'olivetti.pkz')
     if not exists(filepath):
         if not download_if_missing:
             raise IOError("Data not found and `download_if_missing` is False")
 
         print('downloading Olivetti faces from %s to %s'
```

Comment:
There is no consistency to be honest at the moment so I would rather change that in a separate PR. To be frank I was slightly surprised that logging was used inside scikit-learn.